docker-compose.yml

```diff
@@ -14,14 +15,13 @@ x-uber: &uber
     - uber_secret_broker_url=amqp://celery:celery@rabbitmq:5672/uber
     - uber_url_root=http://localhost
     - uber_hostname=localhost
-    - uber_plugins=["magprime"]
     - uber_dev_box=True
   volumes:
     - $PWD:/app/
-    - $PWD/../magprime:/app/plugins/magprime
+    - $PWD/.pythonstartup.py:/app/.pythonstartup.py
```

Comment:
This should be available already as part of `- $PWD:/app/` since it mounts in the whole directory.
```bash
/app # ls -la .*.py
-rw-r--r--    1 root     root          1226 May  7 01:25 .pythonstartup.py
```

Reply:
Huh... I swear my web container didn't have it, but now it does. Not sure how that happened, but I'll remove this line.